extension/src/paymentHandler/manual-capture.handler.js

```diff
@@ -20,6 +20,8 @@ async function execute(paymentObject) {
       currency: chargeInitialTransaction.amount.currencyCode,
     },
     originalReference: authorizationSuccessTransaction.interactionId,
+    reference:
+      chargeInitialTransaction.custom?.fields?.reference || paymentObject.key,
```

Comment:
Did it have any reference before? Maybe we should just stick to the default behaviour and not add the payment key as this was not part of this ticket?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/1011/commits/fc905e630d6ec2b7fe419f037f58ca6b85888b7a